modules/ROOT/pages/api-auto-discovery-new-concept.adoc

```diff
@@ -1,34 +1,35 @@
-= API Gateway API Autodiscovery concepts
+= API Autodiscovery
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-Use API Autodiscovery to pair an API in API Manager to its deployed Mule application.
+Configuring autodiscovery allows a deployed Mule runtime engine (Mule) application to connect with API Manager to download and manage policies and to generate analytics data. Additionally with autodiscovery, your can configure your Mule applications to act as their own API proxy. 
 
-Setting up autodiscovery allows API Manager to manage the API. When set up correctly, it shows the API status, represented as a green dot if the API is being tracked, or a gray dot if untracked.
-
-By configuring autodiscovery you can use policy management, and API Analytics, accessed from within API Manager. +
-When autodiscovery is correctly configured in your Mule application you can say that your application's API is _tracked by_ or _paired to_ API Manager.
-
-An API in a Mule runtime can only be associated with one autodiscovery instance. No two Autodiscoveries can be associated to the same API in a Mule Runtime.
+When autodiscovery is correctly configured in your Mule application, you can say that your application's API is _tracked by_ (green dot) or _paired to_ API Manager. You can associate an API in a Mule setup with only one autodiscovery instance at a given time. 
 
 == Prerequisites
 
-To configure autodiscovery for your Mule application you need to meet the following requirements:
+To configure autodiscovery for your Mule application, ensure that:
 
-* API must exist in API Manager, configured with either a basic endpoint, or a proxy endpoint.
-* Mule must be configured to use Anypoint Platform credentials.
-** The platform credentials give your application access to the API Configuration in API Manager. These credentials must be set before starting the Mule runtime that executes your application. +
-See xref:org-credentials-config-mule4.adoc[Configuring Organization Credentials in Mule Runtime 4] or xref:org-credentials-config-mule3.aroc[Configuring Organization Credentials in Mule Runtime 3] for more details.
-* The autodiscovery element must be configured in your Mule application. This XML element must point to the specific API in API manager which you want to pair to.
+* The API exists in API Manager and is configured as either a basic endpoint, or a proxy endpoint.
+* The Mule application is configured to use Anypoint Platform credentials.
+** The platform credentials give your application access to the API Configuration in API Manager. 
```

Comment:
Since this is the only bullet at this level, it would be better to use a list continuation (+) instead.

Reply:
Done! Thanks! Good catch!